test_data/makedata_suites/060_computed_values.js

```diff
@@ -0,0 +1,821 @@
+/* global print, semver, progress, createCollectionSafe, db, fs, PWD */
+
+
+const { count } = require("console");
+const { stringify } = require("querystring");
+
+(function () {
+  const a = require("@arangodb/analyzers");
+  return {
+    isSupported: function (currentVersion, oldVersion, options, enterprise, cluster) {
+      let currentVersionSemver = semver.parse(semver.coerce(currentVersion));
```

Comment:
you don't use currentVersionSemver, delete

Reply:
Done.